libs/blocks/modal/modal.js

```diff
@@ -146,12 +146,6 @@ export default function init(el) {
   return null;
 }
 
-// Event-based modal
-window.addEventListener('modal:open', (e) => {
```

Comment:
Moving the event listener for the modal causes unit-test failures in `test/blocks/modals/modals.test.js`. Looks like the event listener is not registered when running the modal tests since it now is registered as part of `decorateMeta` in `libs/utils/utils.js`.

Reply:
Thank you for catching. Removed the failing test as it's functionality is accounted for in utils.test.js